sandbox_scripts/QualiEnvironmentUtils/Sandbox.py

```diff
@@ -20,9 +24,10 @@ def __init__(self, reservation_id, logger):
             """:type : logging.Logger"""
             self.api_session = helpers.get_api_session()
             self.id = reservation_id
-
+            self.owner = helpers.get_reservation_context_details().owner_user
```

Comment:
the get_reservation_context_details result should be saved into a variable and then from there, you can take the owner and environment instead of calling this function twice.